packages/jss/src/StyleSheet.js

```diff
@@ -34,7 +31,6 @@ export default class StyleSheet {
   constructor(styles: Object, options: StyleSheetOptions) {
     this.attached = false
     this.deployed = false
-    this.linked = false
```

Comment:
Why did you remove the linked property?

Reply:
because we don't need it any more, if options.link is true, then it is always linked, there is no separate sheet.link() call.

Reply:
Yeah makes sense